internal/kv/mem/mem_kv.go

```diff
@@ -59,6 +79,18 @@ func (kv *MemoryKV) Load(key string) (string, error) {
 	return item.(memoryKVItem).value, nil
 }
 
+// LoadBytes loads an object with @key.
+func (kv *MemoryKV) LoadBytes(key string) ([]byte, error) {
+	kv.RLock()
+	defer kv.RUnlock()
+	item := kv.tree.Get(memoryKVBytesItem{key: key})
+	// TODO，load unexisted key behavior is weird
+	if item == nil {
+		return []byte{}, nil
+	}
+	return clone(item.(memoryKVBytesItem).value), nil
```

Comment:
Looks like this type assertion may panic if we mix using `Save(key, value string)` and `LoadBytes(key string)`